.eslintrc

```diff
@@ -18,7 +18,8 @@
     "no-unused-vars": [
       1,
       {
-        "argsIgnorePattern": "res|next|^err|_"
+        "argsIgnorePattern": "res|next|^err|_",
+        "ignoreRestSiblings": true
```

Comment:
This is to allow creating a copy of an object [excluding certain properties through using rest syntax](https://github.com/eslint/eslint/issues/4880)